src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/ScoringCard.test.jsx

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
 import { shallow } from '@edx/react-unit-test-utils';
+import { mergeConfig } from '@edx/frontend-platform';
 import { formatMessage } from '../../../../../../../testUtils';
 import { scoringCardHooks } from '../hooks';
 import { ScoringCard } from './ScoringCard';
+import { GradingMethodKeys } from '../../../../../../data/constants/problem';
```

Comment:
Same comment about relative import paths as before (I see that `formatMessage` is also doing this here, could be nice to clean that up too)